modules/api_reference/devportal_api_reference.install

```diff
@@ -174,21 +174,26 @@ function devportal_api_reference_update_8002(): ?string {
   // Start a DB transaction in order to have some hopefully meaningful rollback
   // option if anything goes wrong.
   $transaction = $database->startTransaction();
+  $view_id = 'devportal_api_reference';
   try {
-    $view = View::load('devportal_api_reference');
-    // Rename page_1 display to admin_api_reference_list.
-    $displays = $view->get('display');
-    $displays['admin_api_reference_list'] = $displays['page_1'];
-    unset($displays['page_1']);
-    $displays['admin_api_reference_list']['id'] = 'admin_api_reference_list';
-    $view->set('display', $displays);
-    $view->save();
+    if ($view = View::load($view_id)) {
```

Comment:
The new PHPCS configuration from the drupal-qa package is going to throw an error if you define a variable in a condition because you can accidentally override a variable from the outer scope.